SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimPhase2ESProducer.h

```diff
@@ -0,0 +1,81 @@
+
+// user include files
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+
+#include "CondFormats/DataRecord/interface/EcalTPGCrystalStatusRcd.h"
+#include "CondFormats/DataRecord/interface/EcalTPGPhysicsConstRcd.h"
+
+//#include "CondFormats/DataRecord/interface/EcalTPGSpikeRcd.h"
+#include "CondFormats/DataRecord/interface/EcalTPGStripStatusRcd.h"
+//#include "CondFormats/DataRecord/interface/EcalTPGTowerStatusRcd.h"
+
+#include "CondFormats/DataRecord/interface/EcalEBPhase2TPGLinearizationConstRcd.h"
+#include "CondFormats/DataRecord/interface/EcalTPGWeightGroupRcd.h"
+#include "CondFormats/DataRecord/interface/EcalEBPhase2TPGAmplWeightIdMapRcd.h"
+#include "CondFormats/DataRecord/interface/EcalEBPhase2TPGTimeWeightIdMapRcd.h"
+#include "CondFormats/DataRecord/interface/EcalEBPhase2TPGPedestalsRcd.h"
+#include "CondFormats/EcalObjects/interface/EcalTPGCrystalStatus.h"
+#include "CondFormats/EcalObjects/interface/EcalEBPhase2TPGLinearizationConst.h"
+#include "CondFormats/EcalObjects/interface/EcalEBPhase2TPGPedestals.h"
+#include "CondFormats/EcalObjects/interface/EcalTPGPhysicsConst.h"
+//#include "CondFormats/EcalObjects/interface/EcalTPGSpike.h"
+#include "CondFormats/EcalObjects/interface/EcalTPGStripStatus.h"
+//#include "CondFormats/EcalObjects/interface/EcalTPGTowerStatus.h"
+#include "CondFormats/EcalObjects/interface/EcalTPGWeightGroup.h"
+#include "CondFormats/EcalObjects/interface/EcalEBPhase2TPGAmplWeightIdMap.h"
+#include "CondFormats/EcalObjects/interface/EcalEBPhase2TPGTimeWeightIdMap.h"
+
+#include "zlib.h"
+
+/** \class EcalEBTrigPrimPhase2ESProducer
+\author L. Lutton, N. Marinelli - Univ. of Notre Dame
+ Description: forPhase II 
+*/
+
+class EcalEBTrigPrimPhase2ESProducer : public edm::ESProducer {
+public:
+  EcalEBTrigPrimPhase2ESProducer(const edm::ParameterSet &);
+  ~EcalEBTrigPrimPhase2ESProducer() override;
+
+  std::unique_ptr<EcalEBPhase2TPGLinearizationConst> produceLinearizationConst(
+      const EcalEBPhase2TPGLinearizationConstRcd &);
+  std::unique_ptr<EcalEBPhase2TPGPedestalsMap> producePedestals(const EcalEBPhase2TPGPedestalsRcd &);
+  std::unique_ptr<EcalEBPhase2TPGAmplWeightIdMap> produceAmpWeight(const EcalEBPhase2TPGAmplWeightIdMapRcd &);
+  std::unique_ptr<EcalEBPhase2TPGTimeWeightIdMap> produceTimeWeight(const EcalEBPhase2TPGTimeWeightIdMapRcd &);
+  std::unique_ptr<EcalTPGWeightGroup> produceWeightGroup(const EcalTPGWeightGroupRcd &);
+  //std::unique_ptr<EcalTPGLutGroup> produceLutGroup(const EcalTPGLutGroupRcd &);
+
+  std::unique_ptr<EcalTPGPhysicsConst> producePhysicsConst(const EcalTPGPhysicsConstRcd &);
+  std::unique_ptr<EcalTPGCrystalStatus> produceBadX(const EcalTPGCrystalStatusRcd &);
+  //std::unique_ptr<EcalTPGStripStatus> produceBadStrip(const EcalTPGStripStatusRcd &);
```

Comment:
Remove commented out code or explain why it should stay in the comment.

Reply:
Same comment as in the .cc. A comment about the commented lines has been added in the .cc